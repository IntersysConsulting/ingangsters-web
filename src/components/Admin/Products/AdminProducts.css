.bottomWrapper {
  width: 100%;
  display: flex;
  justify-content: space-around;
  max-width: 1300px;
  margin: 25px auto;
}

.newProductWrapper > .btn {
  background-color: var(--light-blue);
  border-color: var(--light-blue);
}
