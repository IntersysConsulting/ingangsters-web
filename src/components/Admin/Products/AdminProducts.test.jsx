import React from "react";
import { shallow } from "enzyme";
import AdminProducts from "./index";
import configureStore from "redux-mock-store";

describe("ADMIN: Products", () => {
  it("renders properly", () => {
    const fakeStore = configureStore()({
      adminProducts: { currentPage: 1, totalItems: 16, itemsPerPage: 10 }
    });
    const render = shallow(<AdminProducts store={fakeStore} dispatch={null} />);
    expect(render).toMatchSnapshot();
  });
});
