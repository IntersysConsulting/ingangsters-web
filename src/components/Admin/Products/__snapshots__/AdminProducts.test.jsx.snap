// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ADMIN: Products renders properly 1`] = `
<ContextProvider
  value={null}
>
  <AdminProducts
    currentPage={1}
    dispatch={null}
    fetchProducts={[Function]}
    itemsPerPage={10}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
    totalItems={16}
  />
</ContextProvider>
`;
